scripts/generate_pinval/generate_pinval.py

```diff
@@ -480,10 +478,13 @@ def main() -> None:
     print("Shape of df_assessment_all:", df_assessment_all.shape)
 
     if df_assessment_all.empty:
-        raise ValueError("No assessment rows returned for the given parameters")
+        raise ValueError(
+            f"No assessment rows returned for the following params: {params_assessment}"
+        )
```

Comment:
While testing these error messages I realized it's clearer if we actually print the parameters.